src/Symfony/Component/Cache/Adapter/RecordingAdapter.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+/*
+ * This file is part of php-cache\cache-bundle package.
+ *
+ * (c) 2015-2015 Aaron Scherer <aequasi@gmail.com>, Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\Component\Cache\Adapter;
+
+use Psr\Cache\CacheItemInterface;
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\Stopwatch\Stopwatch;
+
+/**
+ * An adapter that logs and collects all your cache calls.
+ *
+ * @author Aaron Scherer <aequasi@gmail.com>
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+final class RecordingAdapter implements AdapterInterface
```

Comment:
Oh, I guess this should be named TraceableAdapter fr consistency with the code base (eg TraceableUrlMatcher, etc.)